scripts/install-binary.js

```diff
@@ -40,9 +40,20 @@ const JQ_NAME_MAP = {
   win32: 'jq.exe'
 }
 const JQ_NAME =
-  platform in JQ_NAME_MAP ? JQ_NAME_MAP[platform] : JQ_NAME_MAP.def
+    platform in JQ_NAME_MAP ? JQ_NAME_MAP[platform] : JQ_NAME_MAP.def
 
+const PACKAGE_FOLDER = path.join(__dirname, '..')
+const PACKAGE_FILE = path.join(PACKAGE_FOLDER, 'package.json')
 const OUTPUT_DIR = path.join(__dirname, '..', 'bin')
+const OUTPUT_FILE = path.join(OUTPUT_DIR, JQ_NAME)
+
+const renamePackageBin = () => {
```

Comment:
Can you rename this function to "makeBinaryWorkInWindows"? and add comments on all things happening here? Replace `\` with `/` and also implement the append of ".exe"?

Reply:
I resolved: